tembo-pgmq-python/README.md

```diff
@@ -20,23 +20,56 @@ Postgres running the [Tembo PGMQ extension](https://github.com/tembo-io/tembo/tr
 docker run -d --name postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 quay.io/tembo/pgmq-pg:latest
 ```
 
-### Initialize a connection to Postgres
+### Using Environment Variables
+
+Create a `.env` file in your project root with the following content:
```

Comment:
I don't think we should use python-dotenv in the readme example since we are going for simplicity and minimizing external dependencies. Can we do something like this instead?

Having trouble formatting the nested code block, so apologies for that.

--- 

Set environment variables

```text
export PG_HOST=127.0.0.1
export PG_PORT=5432
export PG_USERNAME=postgres
export PG_PASSWORD=postgres
export PG_DATABASE=test_db
```

PGMQ initialize a connection using the environment variables by default:

```python
from tembo_pgmq_python import PGMQueue, Message

queue = PGMQueue()
```

Reply:
Ok, all do that. I also remove `python-dotenv` from dependencies. but for test of this differences I think we need more details to implement.